specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/examples/PotentialOutages_Get.json

```diff
@@ -0,0 +1,53 @@
+{
+  "parameters": {
+    "potentialOutageName": "default",
+    "api-version": "2021-01-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
```

Comment:
not formatted like an arm resources - missing fields such as id, type, etc.

Reply:
addressed in another PR.